package.json

```diff
@@ -9,7 +9,9 @@
     "@emotion/styled": "^10.0.23",
     "@govtechsg/decentralized-renderer-react-components": "^3.0.4",
     "@hot-loader/react-dom": "^16.11.0",
+    "@types/qrcode.react": "^1.0.1",
     "debug": "4.1.1",
+    "qrcode.react": "^1.0.0",
```

Comment:
this is the exact reason I dont want to add qr code (for instance) into the code base. It must be a dev dependency otherwise that means the code is shipped into production